source/3/3.3/accessories.rst

```diff
@@ -7,4 +7,5 @@ This chapter contains the list of accessories installed by default.
    :maxdepth: 2
    :titlesonly:
 
+   3.1.4/transmission
```

Comment:
This shouldn't be here, it should be in the similar file under 3.1.

Reply:
fixing rn